libcaf_core/caf/detail/profiled_send.hpp

```diff
@@ -46,16 +46,21 @@ void profiled_send(Self* self, Sender&& sender, const Handle& receiver,
 }
 
 template <class Self, class Sender, class Handle, class... Ts>
-void profiled_send(Self* self, Sender&& sender, const Handle& receiver,
+void profiled_send(Self* self, Sender&& sender, const Handle& dst,
```

Comment:
Consistent naming would be cool.
```suggestion
void profiled_send(Self* self, Sender&& src, const Handle& dst,
```
This fits line 35 too.